lib/grape/endpoint.rb

```diff
@@ -248,6 +248,8 @@ def run
 
         run_filters befores, :before
 
+        raise Grape::Exceptions::MethodNotAllowed, header.merge('Allow' => env[Grape::Env::GRAPE_METHOD_NOT_ALLOWED]) if env[Grape::Env::GRAPE_METHOD_NOT_ALLOWED]
```

Comment:
Could you get rid of the duplication of `env[Grape::Env::GRAPE_METHOD_NOT_ALLOWED]` ?